functions.php

```diff
@@ -14,11 +22,117 @@ function getData($db)
     return $plants;
 }
 
-function processData($plants)
+/***
+ *
+ * function checks if the data retrieved from db is
+ * an indexed array and arranges it in to a list
+ * within a div that can be displayed
+ *
+ * @param array $plants
+ * @return string
+ */
+
+function processData(array $plants): string
 {
+
+    if (array_keys($plants) !== range(0, count($plants) - 1)) {
+        return false;
+    }
+
     $plantRow = '';
     foreach ($plants as $plant) {
-        $plantRow .= '<div class="plant">' . $plant['id'] . $plant['common_name'] . $plant['latin_name'] . $plant['identification_image'] . $plant['canopy_level'] . $plant['life_span'];
+        $plantRow .=
+            '<div class="row"><ul><li>'
+            . $plant['id'] . '</li><li>'
+            . $plant['common_name'] . '</li><li>'
+            . $plant['latin_name'] . '</li><li>'
+            . $plant['canopy_level'] . '</li><li>'
+            . $plant['life_span'] . '</li></ul></div>';
     }
     return $plantRow;
-}
\ No newline at end of file
+}
+
+/***
+ *
+ * function retrieves an array of images from the db
+ *
+ * @param PDO $db
+ * @return array
+ */
+
+function getImage(PDO $db): array
+{
+    $sql = $db->prepare('SELECT `identification_image` FROM `Plants`;');
+
+    $sql->execute();
+
+    $images = $sql->fetchAll();
+
+    return $images;
+}
+
+/***
+ *
+ * function checks if data retrieved from the db is
+ * an indexed array and wraps it in a div that can
+ * be displayed
+ *
+ * @param array $images
+ * @return string
+ */
+
+function displayImage(array $images): string {
+
+    if (array_keys($images) !== range(0, count($images) - 1))
+    return false;
```

Comment:
Indent this.
Also, you don't *need* the curlyboys for a one-liner if statement, but they do help readability.